### ISSUE TITLE
release xdress docs on https://readthedocs.org/

### ISSUE THREAD

**Issue by luzfcb** [xdress/xdress#277]
readthedocs currently integrates with github and automatically updates the documentation

**Comment by glyg** [comment]
Hi @luzfcb have you heard from the maintainers? I'm interested in trying out xdress, but if no one is out there maintaining it, it might not be such a good idea...

G.

**Comment by luzfcb** [comment]
I gave up using this project.
anyway if you want to, I just generate documentation in a new repository in readthedocs http://xdress-fabio.readthedocs.org/en/latest/

**Comment by glyg** [comment]
OK, thanks for the feed-back. I saw the website you point at, but am more concerned with maintenance.

G.

**Comment by esafak** [comment]
This is important because [the official site is currently down](http://xdress.org/) (the domain name is expired).

**Comment by scopatz** [comment]
Yeah, someone snipped the domain name while I was travelling and my credit card on that account had expired.  I'll try to get a different domain.

**Comment by empyrical** [comment]
You should still be able to get it again - a whois shows the domain as non-existant
